### PR TITLE
Suggestion from last scrum

### PR DESCRIPTION
**Pull request by hhpatel14**
Updates: 

- Add vsm plugin as a component 
- Update 'ProtectedNamespace' field for VSR
- backup diagram: source -->  from volumeSnapshotContent to cloned volumeSnapshot
- restore diagram: changed "User" image to make it look better 
- changed `VolumeSnapshotMover` to `Data Mover`

### CODE REVIEW

**Comment by mateusoliveira43** [comment]
For review rendered content https://github.com/hhpatel14/oadp-operator/blob/AddVsmPlugin/blogs/data-mover/data-mover-1.2.0-overview.md

File: blogs/data-mover/data-mover-1.2.0-overview.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 <p dir="auto">OADP is the OpenShift API for Data Protection operator. This open source operator sets up and installs Velero on the OpenShift platform, allowing users to backup and restore applications. We will be installing Velero alongside the CSI plugin (modified version).</p>
 
-<h3><a href="https://github.com/openshift/velero-plugin-for-csi/tree/data-mover">CSI PLUGIN (CSI)</a>:</h3>
+<h3><a href="https://github.com/vmware-tanzu/velero-plugin-for-csi">CSI PLUGIN</a>:</h3>
 
-<p dir="auto">The upstream Velero plugin for CSI is modified to facilitate CSI volumesnapshot data movement from an OpenShift cluster to object storage and vice versa.</p>
+<p dir="auto">The collection of Velero plugins for snapshotting CSI backed PVCs using the CSI beta snapshot APIs.</p>
 
 <h3><a href="https://volsync.readthedocs.io/en/stable/">VOLSYNC</a>:</h3>
```
Comment (mateusoliveira43) [inline blogs/data-mover/data-mover-1.2.0-overview.md:29]: I would change the link here to VolSync repo, just to follow what the other links do

File: blogs/data-mover/data-mover-1.2.0-overview.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 
 <p dir="auto">You can read more about CSI<span>&nbsp;</span><a href="https://kubernetes-csi.github.io/docs/">here</a>.</p>
 
-<h2>Why We Need VolumeSnapshotMover</h2>
+<h2>Why We Need Data Mover</h2>
 
 <p dir="auto">During a backup using Velero with CSI, CSI snapshotting is performed. This snapshot is created on the storage provider where the snapshot was taken. This means that for some providers, such as ODF, the snapshot lives on the cluster. Due to this poor durability, in the case of a disaster scenario, the snapshot is also subjected to disaster.</p>
 
-<p dir="auto">With volumeSnapshotMover, snapshots are relocated off of the cluster to the targeted backupStorageLocation (generally object storage), providing additional safety.</p>
+<p dir="auto">With volumeSnapshotMover controller, snapshots are relocated off of the cluster to the targeted backupStorageLocation (generally object storage), providing additional safety.</p>
```
Comment (mateusoliveira43) [inline blogs/data-mover/data-mover-1.2.0-overview.md:17]: I would change backupStorageLocation to backup storage location (or something similar), this seems like a yaml ref, and we did not cite it

Comment (mateusoliveira43) [inline blogs/data-mover/data-mover-1.2.0-overview.md:17]: I would change backupStorageLocation to backup storage location (or something similar), this seems like a yaml ref, and we did not cite it

File: blogs/data-mover/data-mover-1.2.0-overview.md [hunk 3]
```diff
@@ -95,7 +99,7 @@ A VSR represents a restore of a PVC from a snapshot. It can be used to restore a
 </tr>
 <tr>
 <td>ProtectedNamespace</td>
```
Comment (mateusoliveira43) [inline blogs/data-mover/data-mover-1.2.0-overview.md:101]: I would change the order of the table items

Both be 1 - protectedNamespace, 2 - ResticSecretRef, 3 - the last one

**Comment by mateusoliveira43** [comment]
Some other changes I would do (but since you did not touch these parts, we can do this in [Wes' PR](https://github.com/openshift/oadp-operator/pull/1087))

Screenshot from 2023-07-17 10-04-29
would remove the highlighted text, seems just duplication of following explanation of VSR

Screenshot from 2023-07-17 10-04-56
the highlight is wrong, right? it not name (string), it is ref (object) https://github.com/migtools/volume-snapshot-mover/blob/master/config/crd/bases/datamover.oadp.openshift.io_volumesnapshotbackups.yaml#L93

Screenshot from 2023-07-17 10-05-14
The highlighted part confused me. I understand that when a backup is created, immediately after, a restore is created, without user asking to. can we modify wording here? (or I understood wrong?)

File: blogs/data-mover/data-mover-1.2.0-overview.md (hunk 2 above)
Comment (hhpatel14) [reply blogs/data-mover/data-mover-1.2.0-overview.md:17]: it is yaml ref but clarified to an object storage in bracket.

**Review by weshayutin — APPROVED** [review]

**Pull request merged**